src/Microsoft.VisualBasic/src/Microsoft.VisualBasic.vbproj

```diff
@@ -13,6 +13,7 @@
     <AssemblyName>Microsoft.VisualBasic</AssemblyName>
     <NoStdLib>true</NoStdLib>
     <RootNamespace />
+    <Configurations>netstandard-Debug;netstandard-Release;netstandard1.3-Debug;netstandard1.3-Release</Configurations>
```

Comment:
I think to fix this for real, we should add a `*.vbproj` line here:

https://github.com/dotnet/corefx/blob/20ac3b14daf406a98a8a4966ce12c04c9d566cc0/build.proj#L65-L66

And then run `.\build-managed.cmd -- /t:UpdateVSConfigurations`, which will fix this project.  Then going forward, it won't get out of date again.